host/class/uvc/usb_host_uvc/src/descriptor.c

```diff
@@ -730,6 +826,12 @@ static void print_class_specific_desc(const uint8_t *buff)
     case VS_FRAME_MJPEG:
         print_vs_frame_mjpeg_desc(buff);
         break;
+    case VS_FORMAT_FRAME_BASED:
+        print_vs_format_frame_based_desc(buff);
+        break;
+    case VS_FRAME_FRAME_BASED:
+        print_vs_frame_frame_based_desc(buff);
```

Comment:
Nitpick: 
`print_vs_format_frame_based_desc` and `print_vs_frame_frame_based_desc` - very easy to confuse.
Is there any chance to change the name of  `VS_FORMAT_FRAME_BASED` and `VS_FRAME_FRAME_BASED`?

Reply:
This is the standard format in the protocol.
![image](https://github.com/espressif/esp-usb/assets/68437239/1fbcfb2d-ed6d-427f-ba50-b9958b42e05e)